ansible/roles/cuda/tasks/main.yaml

```
@@ -68,15 +52,15 @@
 - name: Add PATH to .bashrc
   ansible.builtin.lineinfile:
     dest: ~/.bashrc
-    line: export PATH="/usr/local/cuda/bin:$PATH"
+    line: export PATH="/usr/local/cuda-12.3/bin:$PATH"
     state: present
     create: true
     mode: 0644
 
 - name: Add LD_LIBRARY_PATH to bashrc
   ansible.builtin.lineinfile:
     dest: ~/.bashrc
-    line: export LD_LIBRARY_PATH="/usr/local/cuda/lib64:$LD_LIBRARY_PATH"
+    line: export LD_LIBRARY_PATH="/usr/local/cuda-12.3/lib64:$LD_LIBRARY_PATH"
```

```suggestion
    line: export LD_LIBRARY_PATH="/usr/local/cuda/lib64:$LD_LIBRARY_PATH"
```
Same as https://github.com/autowarefoundation/autoware/pull/3956/files#r1380347105 above.